vm-zone-move/cloud/azure/azure_helper.py

```diff
@@ -0,0 +1,24 @@
+import random
+from azure.mgmt.compute.v2023_09_01.models import VirtualMachine
+
+class AzureHelper:
+
+    @classmethod
+    def get_zone_for_newvm(cls, azure_old_vm: VirtualMachine, location: str) -> str:
+        account_types_ZRS = ["PREMIUM_ZRS", "STANDARDSSD_ZRS"]
+        #checking if the disk are zrs or not -- all disk need to be zrs for zonal vm otherwise even if one is lrs then logic according to lrs
```

Comment:
"even if one is lrs then logic according to lrs" this sounds a bit incomplete.

Let's change to "check if all disks are ZRS. If yes, new VM will be zonal. Otherwise, if LRS disks are found, new VM will be regional"

Reply:
updated comment